.gitignore

```
@@ -1,7 +1,7 @@
 # Deployment
-.env*
-**/data/certbot/
-**/data/nginx/app.conf
+.env*	
+**/data/certbot/	
+**/data/nginx/app.conf	
```

Why does this keep happening 😆 
```suggestion
.env*
**/data/certbot/
**/data/nginx/app.conf
```